org.eclipse.dawnsci.remotedataset.client/src/org/eclipse/dawnsci/remotedataset/client/RemoteDatasetInfo.java

```diff
@@ -0,0 +1,77 @@
+package org.eclipse.dawnsci.remotedataset.client;
+
+import java.util.List;
+
+import org.eclipse.january.dataset.ShapeUtils;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.eclipse.january.dataset.ShapeUtils'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)